test_scripts/Polices/Policy_Table_Update/021_ATF_P_TC_timeout_countdown_start.lua

```diff
@@ -0,0 +1,203 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- [PolicyTableUpdate] PoliciesManager changes status to “UPDATING”
+-- [HMI API] SystemRequest request/response
+--
+-- Description:
```

Comment:
Please check description it should be more like: 
`PoliciesManager must start timeout taken from "timeout_after_x_seconds" field of LocalPT right after OnSystemRequest is sent out to mobile app`.